project-base/src/Shopsys/ShopBundle/Form/Front/Product/ProductFilterFormType.php

```diff
@@ -32,26 +34,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'currency' => false,
                 'scale' => $priceScale,
                 'required' => false,
-                'attr' => ['placeholder' => $priceTransformer->transform($config->getPriceRange()->getMinimalPrice())],
+                'attr' => ['placeholder' => $viewTransformer->transform($modelTransformer->transform($config->getPriceRange()->getMinimalPrice()))],
```

Comment:
There lines started to be difficult to read and understand (as they are doing a lot of tranformation) please consider extracting into a function or variable

Reply:
We extracted it into a method after the model transformer was automatically added in MoneyTypeExtension